meshapp/src/components/Header.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+    AppBar,
+    Grid,
+    IconButton,
+    Toolbar,
+    Typography,
+}
+    from "@mui/material";
+
+import MenuIcon from '@mui/icons-material/TableRows';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+
+import React from "react";
+
+const Header = () => {
+
+    return (
+        <AppBar position="static" sx={{ background: "#0B7D66" }}>
+
+            <Toolbar>
+                {/*---Left Side Menu Elements (Menu/Notification Icon)---*/}
+                <Grid container>
+                    <IconButton
+                        sx={{ mr: 2, color: "#0000008F" }}
```

Comment:
The color hex ("#0000008F") seems wrong, what's it for?

Reply:
I followed the figma and the icons had that color. 😄